MasterForm.Designer.cs

```diff
@@ -183,325 +183,304 @@ partial class MasterForm {
-            this.homeTab.FlatAppearance.BorderSize = 0;
-            this.homeTab.FlatStyle = System.Windows.Forms.FlatStyle.Flat;
-            this.homeTab.Font = new System.Drawing.Font("Reem Kufi", 9.749999F, System.Drawing.FontStyle.Bold);
-            this.homeTab.Location = new System.Drawing.Point(10, 43);
-            this.homeTab.Name = "homeTab";
-            this.homeTab.Size = new System.Drawing.Size(135, 37);
-            this.homeTab.TabIndex = 2;
-            this.homeTab.Text = "My Mods";
-            this.homeTab.UseVisualStyleBackColor = false;
-            this.homeTab.Click += new System.EventHandler(this.HomeTab_Click);
-            this.homeTab.MouseEnter += new System.EventHandler(this.BtnHoverOn);
-            this.homeTab.MouseLeave += new System.EventHandler(this.BtnHoverOff);
-            // 
-            // createTab
-            // 
-            this.createTab.BackColor = System.Drawing.Color.DarkGray;
-            this.createTab.FlatAppearance.BorderSize = 0;
-            this.createTab.FlatStyle = System.Windows.Forms.FlatStyle.Flat;
-            this.createTab.Font = new System.Drawing.Font("Reem Kufi", 9.749999F, System.Drawing.FontStyle.Bold);
-            this.createTab.Location = new System.Drawing.Point(151, 43);
-            this.createTab.Name = "createTab";
-            this.createTab.Size = new System.Drawing.Size(135, 37);
-            this.createTab.TabIndex = 3;
-            this.createTab.Text = "Create Modpack";
-            this.createTab.UseVisualStyleBackColor = false;
-            this.createTab.Click += new System.EventHandler(this.CreateTab_Click);
-            this.createTab.MouseEnter += new System.EventHandler(this.BtnHoverOn);
-            this.createTab.MouseLeave += new System.EventHandler(this.BtnHoverOff);
-            // 
-            // homePanel
-            // 
-            this.homePanel.BorderStyle = System.Windows.Forms.BorderStyle.FixedSingle;
-            this.homePanel.Controls.Add(this.manualOverride);
-            this.homePanel.Controls.Add(this.selectEnabled_chb);
-            this.homePanel.Controls.Add(this.homeEnabledLabel);
-            this.homePanel.Controls.Add(this.delModpack);
-            this.homePanel.Controls.Add(this.patchButton);
-            this.homePanel.Controls.Add(this.homeNameLabel);
-            this.homePanel.Controls.Add(this.homeSelectLabel);
-            this.homePanel.Controls.Add(this.modListPanel);
-            this.homePanel.Location = new System.Drawing.Point(5, 89);
-            this.homePanel.Name = "homePanel";
-            this.homePanel.Size = new System.Drawing.Size(563, 356);
-            this.homePanel.TabIndex = 4;
+            // betterPBar
+            // 
+            this.betterPBar.Location = new System.Drawing.Point(4, 448);
+            this.betterPBar.Name = "betterPBar";
+            this.betterPBar.Size = new System.Drawing.Size(570, 44);
+            this.betterPBar.TabIndex = 10;

```

Comment:
I'm not sure if the progress bar needs to have a tab index. Isn't this the order in which UI elements are selected when cycling through them via the tab key?

Reply:
Will revert

Reply:
Removed. The tabindex was actually already there, the designer just automatically moved it.